account_invoice_merge/models/account_invoice.py

```diff
@@ -214,7 +214,7 @@ def make_key(br, fields):
         invoice_line_obj = self.env['account.invoice.line']
         # None if purchase is not installed
```

Comment:
Well, with the blame, I only fix PEP8 issues, but the question is more as curiosity. Have you seen the PR status?